src/logging/logging.service.spec.ts

```diff
@@ -6,6 +6,7 @@ import { IConfigurationService } from '@/config/configuration.service.interface'
 
 const mockClsService = jest.mocked({
```

Comment:
Should we assert the logger gets called with a payload including the `safe_app_user_agent` property in the tests on this file? 

Debugging the tests it seems it's currently being called with an object including `"safe_app_user_agent": undefined`, so we need to mock the call  `mockClsService.get` as it's done for `mockClsService.getId` if we want to test it.

Reply:
Good catch! Done in [dd29079](https://github.com/safe-global/safe-client-gateway/pull/731/commits/dd29079d9d7903a021181e430f2b63f5507216c6)